.ci-builds/.azure-exhaustive-tests.yml

```diff
@@ -71,4 +71,34 @@ jobs:
         CC: clang
         CXX: clang++
 #-------------------------------------------------------------------------
-
+# Based on https://unicode-org.github.io/icu/processes/release/tasks/integration.html#test-icu4j-with-only-little-endian-icu4c-data
+- job: ICU4J_with_only_little_endian_ICU4C_data
+  displayName: 'CJ: ICU4J Linux Clang Tests little-endian ICU4C data (Ubuntu 18.04)'
+  timeoutInMinutes: 50
+  pool:
+    vmImage: 'ubuntu-18.04'
```

Comment:
why such an old OS?

Reply:
I keep it as the same as the value of other build in this file above. Moving to a new version should be done in a separate ticket with the rest of the build in this file all together.